sources/osg/DrawElements.js

```diff
@@ -39,6 +39,12 @@ DrawElements.prototype = {
         var gl = state.getGraphicContext();
         gl.drawElements( this.mode, this.count, this.uType, this.offset );
     },
+    drawElements: function ( state, glParam ) {
```

Comment:
it's git diff gone wild. 
That func drawElements not added  added by this push 
(perhaps added glParam to avoid state.getGraphicContext() each call but nothing else)